config.py

```diff
@@ -0,0 +1,25 @@
+"""Flask configuration."""
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+
+class Config:
+    PATH_TO_INITIAL_DATA = path.join(basedir, 'restaurants.json')
+    SECRET_KEY = environ.get('SECRET_KEY')
+    STATIC_FOLDER = 'static'
```

Comment:
try to keep your config minimal, leave only the variables that you use. `SECRET_KEY`, `STATIC_FOLDER` and some others are not used

Reply:
fixed